packages/api/src/trpc.ts

```diff
@@ -111,6 +120,30 @@ const timingMiddleware = t.middleware(async ({ next, path }) => {
   return result;
 });
 
+const isAdmin = async (user: User) => {
+  try {
+    // Define the URL
+    const url = `https://discord.com/api/v10/guilds/${env.KNIGHTHACKS_GUILD_ID}/members/${user.discordUserId}`;
+
+    // Make the request
+    // eslint-disable-next-line
```

Comment:
Avoid disabling eslint rules as much as possible. If you absolutely need to, please provide a reason like:

```
// eslint-disable-next-line -- INSERT REASON HERE
```

Reply:
Heard